docs/insomnia/requests.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: article-detail
+title:  Requests
+category: "Requests and Responses"
+category-url: requests-and-responses
+---
+
+Insomnia supports sending requests via [HTTP method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods), [gRPC](https://grpc.io/docs), and [GraphQL](https://graphql.org).
+
+To learn how to repurpose parts of your request, see
+[Environment Variables](/insomnia/environment-variables).
+
+## Send an HTTP method request
+
+Send an HTTP method request in a Document or Collection. The following instructions
+assume you're starting with an empty Document or Collection.
+
+In addition to standard HTTP methods, Insomnia supports [Custom HTTP Methods](#custom-http-methods).
+
+1. In a Document, select the **Debug** tab, then **New Request**. In a Collection, select **New Request**.
+1. In the **New Request** modal:
+   1. Enter a request name.
+   1. Select a method from the dropdown.
+   1. Click **Create**.
+1. Enter the API URL in the request URL input.
+1. To add authentication, go to the **Auth** tab. Use the **Auth** dropdown to select your authentication method.
+1. When you have entered your request URL and authentication, click **Send**.
+
+See [Request Options](#request-options) to customize your request.
+
+## Send a gRPC request
+
+Send a gRPC method request in a Document or Collection. The following instructions
+assume you're starting with an empty Document or Collection.
+
+1. In a Document, select the **Debug** tab, then **New Request**. In a Collection, select **New Request**.
+1. In the **New Request** modal:
+   1. Enter a request name.
+   1. Select **gRPC** from the dropdown.
+   1. Click **Create**.
+1. You'll be taken to a **Select Proto File** modal.
+1. Click **Add Directory** or **Add Proto File**.
+1. Add your directory or upload your Proto File. Insomnia will automatically detect the streaming types and modify the first tab as such.
+1. Enter your API URL and click **Start**.
+
+See [gRPC Request Options](#grpc-request-options) to customize your request.
+
+### gRPC Request Options
+
+Modify any gRPC request with the following options.
+
+{:.table .table-striped}
+Option | Description
+------ | ------------
+*Streaming Type* <small>Example: `Unary`</small> | The body of the request. The tab displays the streaming type, auto-detected from the Proto File or directory.<br><br>Streaming types are: Unary, Server Streaming, Client Streaming, and Bidirectional Streaming.
+Headers | Add a header to your request. For example, `Content-Type: application/grpc`.
```

Comment:
The option currently says "Headers", but it should say "Metadata". What we're doing is giving the option for users to add [Metadata](https://grpc.io/docs/what-is-grpc/core-concepts/#metadata) to the request.

We could also link to the gRPC docs for the [streaming type](https://grpc.io/docs/what-is-grpc/core-concepts/#rpc-life-cycle) and the Metadata, what do you think?

```suggestion
Headers | Add gRPC Metadata to your request. For example, `Content-Type: application/grpc`.
```

Reply:
I'm going to stick with "Headers" since that's what the UI has. However, we can change it in the UI if you think that makes sense. Let me know and we can create a Linear task.

<img width="1167" alt="Screen Shot 2022-01-03 at 10 39 08 AM" src="https://user-images.githubusercontent.com/22301405/147967074-7f94a022-7511-43d3-b423-f81102c12060.png">